exspy/models/edsmodel.py

```diff
@@ -827,9 +853,28 @@ def calibrate_xray_lines(
         """
 
         if calibrate == "energy":
-            bound = (bound / eV2keV) * self.units_factor
-            free = self.free_xray_lines_energy
-            fix = self.fix_xray_lines_energy
+            # For energy calibration, reset line energies to reference values from the database
```

Comment:
Maybe update the docstring? The behaviour for energy calibration will be different from other calibration.